lib/rpmvs.c

```diff
@@ -305,13 +306,30 @@ const char *rpmsinfoDescr(struct rpmsinfo_s *sinfo)
 char *rpmsinfoMsg(struct rpmsinfo_s *sinfo)
 {
     char *msg = NULL;
+    char *fphex = NULL;
+    char *fpmsg = NULL;
+    if (sinfo->rc == RPMRC_OK && sinfo->key) {
+	uint8_t *fp;
+	size_t fplen;
```

Comment:
Initialize to NULL and zero respectively.

The rule simply is: NEVER introduce uninitialized variables, they are nothing trouble.

Reply:
Done.